nexus/db-model/src/instance.rs

```diff
@@ -104,7 +110,9 @@ impl Instance {
             ncpus: params.ncpus.into(),
             memory: params.memory.into(),
             hostname: params.hostname.to_string(),
-            boot_on_fault: false,
+            // TODO(eliza): allow this to be configured via the instance-create
+            // params...
```

Comment:
Might be worth a tracking issue, though you might also have this change on deck already. Your call.

Reply:
I've already  gone and added that in #6503  :)